apps/gateway/src/lib/prompt-tokens.spec.ts

```
@@ -0,0 +1,79 @@
+				return promptTokenCount;
+			}
+
+			// Calculate prompt tokens if missing or 0
+			try {
+				// Simple estimation fallback (as in our implementation)
+				const totalChars = messages.reduce(
+					(acc, m) => acc + (m.content?.length || 0),
+					0,
+				);
+				return Math.max(1, Math.round(totalChars / 4));
+			} catch (error) {
```

ESLint

[error] 'error' is defined but never used. Allowed unused caught errors must match /^_/u.

(no-unused-vars)